### DISCUSSION TITLE
输入框翻译不好触发

### DISCUSSION THREAD

**Discussion by kevinbin** [immersive-translate/immersive-translate#1765] [Help]
### 概述 | Summary

在mac chrome浏览器中使用输入框翻译连续3次空格，触发翻译概率低。不知道什么原因

**Comment by keyboard3** [comment]
触发这个主意亮点，一个是要在带网址的网页中触发，另一个是需要空格按住的快一点。 比如去www.bing.com,www.google.com去尝试一下
